.travis.yml

```diff
@@ -38,18 +38,34 @@ jobs:
       env:
       - GROUP=1
       - NEW_RESOLVER=1
+    - env:
```

Comment:
To add on to this, because of the code that we're touching, these are the specific integration tests that would give us the most impact:

1. `pip wheel`
2. `pip download`
3. `pip install` of a wheel that has extras that are also wheels

One approach would be to find existing integration tests, then convert them to use our mock server and then parameterize which mock server to use: plain one or range-request-supporting one.